privval/signer_listener_endpoint.go

```diff
@@ -123,6 +123,9 @@ func (sl *SignerListenerEndpoint) SendRequest(request privvalproto.Message) (*pr
 		return nil, err
 	}
 
+	// Reset pingTimer to avoid sending unnecessary pings.
+	sl.pingTimer.Reset(sl.pingInterval)
```

Comment:
This needs to stop and drain the timer - from the docs:

>Reset should be invoked only on stopped or expired timers with drained channels. If a program has already received a value from t.C, the timer is known to have expired and the channel drained, so t.Reset can be used directly. If a program has not yet received a value from t.C, however, the timer must be stopped and—if Stop reports that the timer expired before being stopped—the channel explicitly drained

Reply:
Nevermind. I thought this was a timer, since it's named, you know, `pingTimer`. :) Resetting tickers is fine.